src/sql/src/plan/statement/ddl.rs

```diff
@@ -1259,6 +1260,9 @@ fn kafka_sink_builder(
         Some(Value::String(topic)) => Some(topic),
         Some(_) => bail!("consistency_topic must be a string"),
     };
+    if consistency_topic.is_some() && consistency.is_some() {
+        bail!("cannot specify consistency_topic and CONSISTENCY TOPIC simultaneously");
+    }
 
```

Comment:
We should do something with `consistency` here right? Are you planning to add that to this PR? If not, probably worth dropping an `unsupported!` here?